HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+v.0.16.5
```

Comment:
Oh, and please replace "v.0.16.5" with "Latest". The label is changed to the former
only before a release.